jelenanemcic/code-cadets-2021
# Homework3

To test the application:
- Initialize databases
- Run controller and calculator
- Run betgenerator script
- Run eventsettler script

I added log messages in controller and in calculator when we update the database according to the new event update, so that we can see that the database gets updated correctly.
I also commented out a few lines in controller that I thought were not needed.
